src/gui/qgscolorwidgets.cpp

```diff
@@ -1017,14 +1019,14 @@ void QgsColorRampWidget::paintEvent( QPaintEvent *event )
     QColor color = QColor( mCurrentColor );
     color.setAlpha( 255 );
     QPen pen;
-    pen.setWidth( 0 );
+    pen.setWidthF( devicePixelRatioF() );
```

Comment:
is this an error? I fail to see the point.

Reply:
This is how it looks like with setWidth( 0 ) and rest of the changes the same 
<img width="320" alt="screenshot 2019-01-28 at 16 10 23" src="https://user-images.githubusercontent.com/804608/51845176-4deb5880-2317-11e9-9aa3-380f76482234.png">